README.md

```diff
@@ -78,6 +78,8 @@ Patches and issues welcome! See [CONTRIBUTING](https://github.com/webrtc/samples
 
 ### Video chat ###
 
+(source code is at https://github.com/webrtc/apprtc)
```

Comment:
The markdown format for links is `[Link text](link)`.

Also I think it would be clearer of you added it to the same line as the AppRTC link like this:

[AppRTC video chat client](https://apprtc.appspot.com/) powered by Google App Engine ([source](https://github.com/webrtc/apprtc))

or 

[AppRTC video chat client](https://apprtc.appspot.com/) powered by Google App Engine (source can be found [here](https://github.com/webrtc/apprtc))

or 

[AppRTC video chat client](https://apprtc.appspot.com/) powered by Google App Engine (source [here](https://github.com/webrtc/apprtc))